zeebe/backup-stores/azure/src/main/java/io/camunda/zeebe/backup/azure/AzureBackupStore.java

```diff
@@ -59,13 +59,23 @@ public AzureBackupStore(final AzureBackupConfig config, final BlobServiceClient
     executor = Executors.newVirtualThreadPerTaskExecutor();
     final BlobContainerClient blobContainerClient = getContainerClient(client, config);
 
-    fileSetManager = new FileSetManager(blobContainerClient);
+    fileSetManager = new FileSetManager(blobContainerClient, config.createContainer());
     manifestManager = new ManifestManager(blobContainerClient, config.createContainer());
   }
 
   public static BlobServiceClient buildClient(final AzureBackupConfig config) {
     // BlobServiceClientBuilder has their own validations, for building the client
-    if (config.connectionString() != null) {
+    if (config.accountSasToken() != null) {
+      return new BlobServiceClientBuilder()
+          .sasToken(config.accountSasToken())
+          .endpoint(config.endpoint())
```

Comment:
Hi @rodrigo-lourenco-lopes ,
would this implementation works?
If we deconstruct the AccountSasToken, it contains the sas token and the Resource URL.
Example from my SAS Token URL
```
https://minhstbackup.blob.core.windows.net/zeebe-backup?sp=r&st=2025-04-29T13:37:04Z&se=2025-04-29T21:37:04Z&spr=https&sv=2024-11-04&sr=c&sig=sig
```
Endpoint also contains the URL, so it will be duplicated.

```suggestion
          .endpoint(config.SasTokenUrl())
```
This should be the right configuration according to the [Documentation Blob Service SAS Token](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/storage/azure-storage-blob/README.md#create-a-blobcontainerclient)

Also to be discussed, when a client wants to test it, the Term SAS Account Token would be confusing. I think SAS Token Url would be more fitting if we look into azure and its ui. 
![image](https://github.com/user-attachments/assets/7f8b3058-1ca2-4e4a-a6bb-41d4c1b1f819)